src/aten/UpSampleNearest1d.cpp

```diff
@@ -0,0 +1,85 @@
+#include <ATen/ATen.h>
+#include <ATen/XPUNativeFunctions.h>
+#include <ATen/core/Tensor.h>
+#include <aten/sycl/UpSampleNearest1d.h>
+
+namespace at {
+// using namespace at::native;
+Tensor& upsample_nearest_meta(
```

Comment:
Not aligned?
<img width="653" alt="image" src="https://github.com/intel/torch-xpu-ops/assets/28250760/c2ecf11c-986d-43d7-9761-b139ae0a3b14">